api/docs/release.dox

```diff
@@ -128,6 +128,14 @@ The changes between version \DR_VERSION and 11.0.0 include the following compati
 changes:
  - Added X64 Linux support to dr_create_memory_dump(). This API has the same
    restriction as dr_suspend_all_other_threads_ex().
+ - Removed uncompleted instructions and the corresponding memrefs from drmemtraces.
+   Instructions which are fetched but not completed due to asynchronous signal or
+   fault are removed. A new marker
+   #dynamorio::drmemtrace::TRACE_MARKER_TYPE_UNCOMPLETED_INSTRUCTION was added to indicate an
+   uncompleted instruction was removed. The value of the marker is the encoding of the
+   removed instruction up to a pointer's length. Added
+   OFFLINE_FILE_VERSION_RETIRED_INSTRUCTIONS_ONLY file type for drmemtraces with
```

Comment:
This is not a "file type" as that sounds like the offline_file_type_t bitfield.  You increased the trace version to this new version to indicate traces where these are removed.